braindecode/util.py

```diff
@@ -355,6 +355,12 @@ def update_estimator_docstring(base_class, docstring):
     return out_docstring
 
 
+def update_moabb_docstring(base_class, docstring):
```

Comment:
```suggestion
def _update_moabb_docstring(base_class, docstring):
```
please make this function private